src/components/auth/ForgotPassword.js

```diff
@@ -1,60 +1,146 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import ForgotPasswordStyled from './ForgotPasswordStyled';
 // import { authPass } from "../../state/auth/authActionCreator"
-// import { connect, useDispatch } from "react-redux"
+import { useDispatch, useSelector } from 'react-redux';
 // import { validateEmail } from "./utils"
 // import { Link } from "react-router-dom"
+import { Formik, Field, Form, ErrorMessage } from 'formik';
+import { message, notification } from 'antd';
+import * as Yup from 'yup';
+import ForgotPasswordImage from '../assets/image/auth/forgot_password.png';
```

Comment:
Codacy found an issue: ['ForgotPasswordImage' is defined but never used.](https://app.codacy.com/gh/codeclannigeria/codeclannigeria-frontend/pullRequest?prid=6093077)